ALUM/ALUM/Components/ProgressBarComponent.swift

```diff
@@ -5,14 +5,15 @@
 //  Created by Yash Ravipati on 1/25/23.
 //
 
-// nodeList - [completed, not completed, completed...]
-
 import SwiftUI
 
+let circleDiameter = CGFloat(16)
+let borderWidth = CGFloat(3)
+
 struct ProgressBarComponent: View {
-    @State var nodes = 6
-    @State var filledNodes = 3
-    @State var activeNode = 3
+    @State var nodes = 3
```

Comment:
Is there a way to change these values so that we can create progress bars with different node lengths? For example, in some places, we may need a progress bar with 4 nodes and in other places, we may only need 3. Not sure if this already exists given my limited knowledge of swift.

Reply:
Great point! Yes, in Swift you can pass in state to View. Added an example of it in the preview section so you can check that out